infra.bs

```diff
@@ -569,6 +569,29 @@ the <a>string</a> with their corresponding <a>code point</a> in <a>ASCII upper a
 <a>ASCII lowercase</a> of <var>B</var>.
 <!-- TODO: define string equals? -->
 
+<p>To <dfn export>ASCII encode</dfn> a <a>string</a> <var>input</var>, run these steps:</p>
```

Comment:
Nit: remove `</p>` here and below. (I know the rules are inconsistent between HTML and other specs -_-.)

Reply:
Whoops, thanks.